.github/workflows/ci-pr.yml

```
@@ -113,7 +114,10 @@ jobs:
         run: pnpm install --frozen-lockfile
 
       - name: Run database migrations
-        run: cd server && npx tsx scripts/run-migrations.ts
+        run: cd server && npm run migrate
+        env:
+          ALLOW_BREAKING_MIGRATIONS: 'true'
+          CI: true
         env:
```

YAMLlint (1.38.0)

[error] duplication of key "env" in mapping

(key-duplicates)